packages/coverage-v8/src/provider.ts

```diff
@@ -443,16 +438,11 @@ export class V8CoverageProvider extends BaseCoverageProvider implements Coverage
     const filePath = normalize(fileURLToPath(url))
 
     let isExecuted = true
-    let transformResult:
-      | FetchResult
-      | Awaited<ReturnType<typeof this.ctx.vitenode.transformRequest>>
-      = transformResults.get(filePath)
+    let transformResult: FetchResult | TransformResult | undefined = transformResults.get(filePath)
 
     if (!transformResult) {
       isExecuted = false
-      transformResult = await this.ctx.vitenode
-        .transformRequest(filePath)
-        .catch(() => null)
+      transformResult = await onTransform(removeRoot(url, FILE_PROTOCOL)).catch(() => undefined)
```

Comment:
Sorry for being pedantic about this, but the function is called `removeRoot` 🤔 This feels a bit confusing to me that you pass down file protocol then 🤔

Reply:
No worries! Any ideas for better name? `removeStartsWith`, `trimStartsWith`, `removeIfStartsWith`?

Reply:
`removeStartsWith` sounds fine to me